src/Features/Core/Portable/LanguageServices/SymbolDisplayService/AbstractSymbolDisplayService.AbstractSymbolDescriptionBuilder.cs

```diff
@@ -487,7 +488,7 @@ private async Task<ImmutableArray<SymbolDisplayPart>> GetFieldPartsAsync(IFieldS
                     }
                 }
 
-                return ToMinimalDisplayParts(symbol, MinimallyQualifiedFormatWithConstants);
+                return ToMinimalDisplayParts(symbol, MinimallyQualifiedFormatWithConstantsAndModifiers);
```

Comment:
major design question for me would be: should we not use this for other members as well...  @sharwell want to take to design review?

Reply:
➡️ Explicitly covered in https://github.com/dotnet/roslyn/issues/26947#issuecomment-390831445